src/reconstruction/enzyme_constrained.jl

```diff
@@ -29,20 +29,35 @@ ecmodel = make_enzyme_constrained_model(
         "total" => 0.5,
     ),
 )
-```
 
+ecmodel2 = make_enzyme_constrained_model(
+    model;
+    total_gene_product_mass_bound = 0.5
+)
+```
 # Notes
 Reactions with no turnover number data, or non-enzymatic reactions that should
-be ignored, must have `nothing` in the `gene_associations` field of the
+be ignored, *must* have `nothing` in the `gene_associations` field of the
 associated reaction.
```

Comment:
gene associations is a _field_? this only works on ObjectModels?

Reply:
ah, well you need to have an accessor implemented to access this, so only objectmodel would work, but you could do this for any other model, fixing the wording

Reply:
btw subsectioning the documentation isn't very good, this isn't "Notes" but "If you didn't read this, you're f*ed, haha!". Preferably put this into description of the thing; it's a normal "input".